src/main/java/teammates/ui/webapi/DeleteStudentsAction.java

```diff
@@ -27,9 +27,10 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
     @Override
     public JsonResult execute() {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String limit = getNonNullRequestParamValue(Const.ParamsNames.LIMIT);
```

Comment:
Can we make it Nullable so that we can still delete student without limit?

Reply:
Good point, but for now I do not see any place where deleting student without limit is beneficial 🤔